README.md

```diff
@@ -1 +1,179 @@
+        <td>Set to false to disable X-Pack security</td>
+      </tr>
+      <tr>
+        <td>xpack.monitoring.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack monitoring</td>
+      </tr>
+      <tr>
+        <td>xpack.graph.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack graph</td>
+      </tr>
+      <tr>
+        <td>xpack.watcher.enabled</td>
+        <td>false</td>
+        <td>Set to false to disable X-Pack watcher</td>
+      </tr>
+    </table> 
+	For more information on X-Pack:
+	* https://www.elastic.co/guide/en/x-pack/current/xpack-settings.html
+	* https://www.elastic.co/guide/en/x-pack/current/installing-xpack.html#xpack-enabling
+
+4. Deploy the services  
+First navigate to the folder where you have downloaded the files to and then run one of the following commands, depending on whether you are using Docker Compose or Docker Stack:
+
+    <table>
+      <tr>
+        <td><b>Docker Compose</b></td>
+        <td>
+			docker-compose up  <small>(docker-compose defaults to use a file called <i><b>docker-compose.yml</b></i>)</small><br />
+			docker-compose up -d <small>(<i><b>-d</b></i> flag is for "detached mode" i.e. run containers in the background)</small>
+		</td>
+      </tr>
+      <tr>
+        <td><b>Docker Stack</b></td>
+        <td>docker stack deploy --compose-file=docker-compose.yml auditservicedemo</td>
+      </tr>
+    </table>
+
+5. Check the Health of Elasticsearch  
+The health of the Elasticsearch container and / or cluster can be inspected by issuing the following command:    
+	`curl http://localhost:9200/_cat/health`    
+	`1493041686 13:48:06 docker-cluster green 2 2 14 7 0 0 0 0 - 100.0%`
+
+6. Index a simple customer document 
+	`curl -XPUT 'localhost:9200/customer/external/1?pretty&pretty' -H 'Content-Type: application/json' -d'
+	{
+  		"name": "John Doe"
+	}'`
+    
+	Reponse:
+    
+	`{
+         "_index" : "customer",
+         "_type" : "external",
+         "_id" : "1",
+         "_version" : 1,
+         "result" : "created",
+         "_shards" : {
+             "total" : 2,
+             "successful" : 2,
+             "failed" : 0
+         },
+         "created" : true
+    }`
+
+7. Retrieve a simple customer document  
+	`curl -XGET 'localhost:9200/customer/external/1?pretty&pretty'`  
+  
+	Response:  
+  
+	`{
+	  "_index" : "customer",
+	  "_type" : "external",
+	  "_id" : "1",
+	  "_version" : 1,
+	  "found" : true,
+	  "_source" : { "name": "John Doe" }
+	}`
+
+### Troubleshooting
+
+#### Errors during Start Up
+
+If the following error appears during start up and the Elasticsearch container shuts down shortly after start up:
+> elasticsearch1  | ERROR: bootstrap checks failed  
+> elasticsearch1  | max virtual memory areas vm.max_map_count [65530] is too low, increase to at least [262144]
+
+This error can be resolved by issuing the following command on the Linux host:  
+* `sudo sysctl -w vm.max_map_count=262144`
```

Comment:
We should remember that we are only trying to set up a developer instance here - not a production instance.  I understand you'd want this in production but we should look again at whether there are settings we can set to reduce the memory requirement for an ease of deployment instance.